public_dropin_gpu_environments/python311_genai/env_info.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "666b39a88c596d95eddb9f26",
+  "name": "[DataRobot][NVIDIA] Python 3.11 GenAI",
+  "description": "This template environment can be used to create GenAI-powered custom models and includes common dependencies for workflows using OpenAI, Langchain, vector DBs, or transformers in PyTorch. This environment is expected to be run on machines with GPU. Similar to other drop-in environments, you can either include a .pth artifact or any other code needed to deserialize your model, and optionally a custom.py file.",
```

Comment:
@jendavies How best rewrite following:  "This environment is expected to be run on machines with GPU"